bin/build-helper

```diff
@@ -32,10 +44,18 @@ buildImage() {
 
 pushImages() {
   createImagesFileIfNotExists
-  cat /dev/null > $BASE_DIR/dist/images.txt
+
+  if [[ $REGISTRY == "" ]]; then
+    echo "REGISTRY env var is not set. Please add REGISTRY to env or as a var before the command."
+    exit 1
+  else
+    formatRegistry
```

Comment:
I don't think this needs to be here - `formateRegistry` gets called in the `buildImage` function before writing to `images.txt`

Reply:
I think you are correct sir! I had originally planned to validate the content of the images.txt file.

Reply:
fixed.